packages/desktop/src/renderer/app/components/environment-routes/environment-routes.component.html

```diff
@@ -447,6 +447,18 @@
             />
           </div>
 
+          <ng-container [formGroup]="activeRouteResponseForm">
```

Comment:
You are already in the same formGroup (see line 381). You can remove this wrapper ng-container